src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -61,7 +61,51 @@ Then, you can see the error in your dashboard:
 [{% asset js-index/error-message.png alt="Error in Unresolved Issues with title This is my fake error message" %}]({% asset js-index/error-message.png @path %})
 
 ## Capturing Errors
-In most situations, you can capture errors automatically with `captureException()`.
+Automatically and manually capture errors, exceptions, and rejections.
+
+### Automatically Capture Errors
+By including and configuring Sentry, the SDK will automatically attach global handlers to capture uncaught exceptions and unhandled rejections.
+
+[{% asset js-index/automatically-capture-errors.png alt="Stack trace of a captured error" %}]({% asset js-index/automatically-capture-errors.png @path %})
+
+Browsers take security measures when serving script files from different origins. To ensure errors are routed to Sentry, configure the CORS headers, and add script attributes. 
+
+**CORS Attributes and Headers**
+
+To gain visibility into a JavaScript exception thrown from scripts originating from different origins, do two things:
+
+1. Add a `crossorigin=”anonymous”` script attribute
+
+    ```javascript
+    <script src="http://another-domain.com/app.js" crossorigin="anonymous"></script>
+    ```
+
+    The script attribute tells the browser to fetch the target file "anonymously." Potentially user-identifying information like cookies or HTTP credentials won't be transmitted by the browser to the server when requesting this file.
+
+2. Add a Cross-Origin HTTP header
+
+    ```javascript
+    Access-Control-Allow-Origin: *
+    ```
+    Cross-Origin Resource Sharing (CORS) is a set of APIs (mostly HTTP headers) that dictate how files ought to be downloaded and served across origins.
+    
+    By setting `Access-Control-Allow-Origin: *`, the server is indicating to browsers that any origin can fetch this file. Alternatively, you can restrict it to a known origin you control:
+    
+    ```javascript
+    Access-Control-Allow-Origin: https://www.example.com
+    ```
+    
+    Most community CDNs properly set an Access-Control-Allow-Origin header.
+
+    ```bash
+    $ curl --head https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.js | \
+    grep -i "access-control-allow-origin"
+
+    Access-Control-Allow-Origin: *
+    ```
+    
```

Comment:
@benvinegar the two steps above were taken from a [blog post](https://blog.sentry.io/2016/05/17/what-is-script-error#the-fix-cors-attributes-and-headers). I've omitted a couple of the ending sentences because they reference an example earlier in the blog post that I didn't add here. Does this section still make sense without those sentences? Here's what I omitted:

> Once you've completed these steps, any errors triggered by this script will report to window.onerror just like any regular same-domain script. So instead of "Script error," the onerror example from the beginning will yield:
> `"ReferenceError: bar is not defined", "http://another-domain.com/app.js", 2, 1, [Object Error]`

Reply:
@PeloWriter will you review lines 73 through 105? Thanks 😄